pkgs/test_core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.3.2-dev
```

Comment:
you can move this into the previous `-dev` - we don't publish `-dev` releases just use that as a marker for the next release